proto/lighthouse-result.proto

```diff
@@ -127,15 +127,18 @@ message LighthouseResult {
   // Message containing the configuration settings for the LH run
   message ConfigSettings {
 
-    // The possible form factors an audit can be run in
-    enum EmulatedFormFactor { // Deprecated
+    // The possible form factors an audit can be run in.
+    // This enum served the emulated_form_factor field, but in v7, that field was deprecated.
+    // Meanwhile in v7, the form_factor field was added and has a mobile/desktop enum, so this
+    // "legacy-ish" enum gets reused.
+    // https://github.com/GoogleChrome/lighthouse/blob/master/docs/emulation.md#changes-made-in-v7
```

Comment:
does any of this matter to consumers of the proto? It's an enum that's semantically used for the same thing as before except `none` was deprecated as an option, and that's the part that's _not_ explained by the comment :P

Maybe (some version of) this comment makes more sense down on `emulated_form_factor` or the new `form_factor`?

Reply:
> does any of this matter to consumers of the proto?

Not really. A LR-like client might be confused that when constructing a request and specifying something like `FormFactor: lhpb.ConfigSettings.EmulatedFormFactor.mobile`.

But that's not really an issue.

> ... except `none` was deprecated as an option, and that's the part that's _not_ explained by the comment :P

I added a small bit on that.

> Maybe (some version of) this comment makes more sense down on `emulated_form_factor` or the new `form_factor`?

Instead i just moved these 3 things to live closer. like a family. <3